package/MDAnalysis/analysis/align.py

```diff
@@ -497,79 +570,38 @@ class AlignTraj(AnalysisBase):
     rmsd : Array
         Array of the rmsd values of the least rmsd between the mobile_atoms
         and reference_atoms after superposition and minimimization of rmsd
-    filename : string
+    filename : str
         String reflecting the filename of the file where mobile_atoms positions
         will be written to upon running RMSD alignment
+
+
+    Notes
+    -----
+    - If set to ``verbose=False``, it is recommended to wrap the statement in a
+      ``try ...  finally`` to guarantee restoring of the log level in the
+      case of an exception.
+    - The ``in_memory`` option changes the `mobile` universe to an
+      in-memory representation (see :mod:`MDAnalysis.coordinates.memory`)
+      for the remainder of the Python session. If ``mobile.trajectory`` is
+      already a :class:`MemoryReader` then it is *always* treated as if
+      ``in_memory`` had been set to ``True``.
+
+
+    .. versionchanged:: 0.16.0
+       new general ``weights`` kwarg replace ``mass_weights``
+
+    .. deprecated:: 0.16.0
+       Instead of ``mass_weighted=True`` use new ``weights='mass'``
+
+    .. versionchanged:: 0.17.0
+       removed deprecated `mass_weighted` keyword
+
     """
 
     def __init__(self, mobile, reference, select='all', filename=None,
-                 prefix='rmsfit_', mass_weighted=None, weights=None,
+                 prefix='rmsfit_', weights=None,
                  tol_mass=0.1, strict=False, force=True, in_memory=False,
                  **kwargs):
-        """Initialization
```

Comment:
The problem with documenting parameters in `__init__()` is that this makes it harder to write numpy-doc conforming docs where the parameters come early and then notes and examples come later. If I want to do this then I can only write a small short blurb in the class docs and then have to do *all* notes and examples in `__init__` after the parameters. 

I would prefer to *not* document `__init__` at all and put everything in the class docs because (1) you have all the relevant class docs in one place and (2) even for derived classes without `__init__` you still have the docs in the same place. 

I will undo my changes here and we should discuss how we want to do this.